.github/workflows/security.yml

```diff
@@ -0,0 +1,65 @@
+name: Security
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+env:
+  CARGO_TERM_COLOR: always
+  CARGO_REGISTRIES_CRATES_IO_PROTOCOL: sparse
+  RUST_VERSION_NEW: "stable"
```

Comment:
Almost all, upgraded with background is bad for CI, because in changes without commit and difficult to debug.